stable/drone/values.yaml

```diff
@@ -248,6 +253,11 @@ agent:
   ##
   nodeSelector: {}
 
+  ## Node taints to tolerate
```

Comment:
Why not add tolerations in templates/deployment-agent.yaml if you include this in the agent values?

Reply:
https://github.com/helm/charts/blob/master/stable/drone/templates/deployment-agent.yaml#L37
already added, but the values.yaml is not updated.

Reply:
fair enough :)